// signaling.proto

syntax = "proto3";

package signaling;

service signaling_service {
	rpc offer(offer_request) returns (offer_response);
}

message offer_request {
	string sdp = 1;
}

message offer_response {
	string sdp = 1;
}